addons/binding/org.openhab.binding.onebusaway/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+    <config-description uri="bridge-type:onebusaway:api-config">
+        <parameter name="apiKey" type="text" required="true">
+            <label>API Key</label>
+            <description>The OneBusAway API key to use.</description>
+        </parameter>
+        <parameter name="apiServer" type="text" required="true">
```

Comment:
I could attempt to construct a list, but the various agencies put this information in different places, unfortunately.